packages/rocketchat-ui-message/client/popup/messagePopup.js

```diff
@@ -268,21 +270,26 @@ Template.messagePopup.onDestroyed(function() {
 });
 
 Template.messagePopup.events({
+	'scroll .rooms-list__list'() {
+		lazyloadtick();
+	},
 	'mouseenter .popup-item'(e) {
 		if (e.currentTarget.className.indexOf('selected') > -1) {
 			return;
 		}
 		const template = Template.instance();
 		const current = template.find('.popup-item.selected');
 		if (current != null) {
-			current.className = current.className.replace(/\sselected/, '').replace('sidebar-item__popup-active', '');
+			current.className = current.className
```

Comment:
Please, do not introduce unnecessary changes

Reply:
it was the auto fix... I think we should accept because the auto linter will try lint each time somebody save the file (: